packages/react-app/src/components/pages/Bounties/Bounty/index.tsx

```diff
@@ -223,26 +220,43 @@ const BountyModal = ({
   onClose: () => void;
 }): JSX.Element => {
 	// TODO Need a Delete action if Draft or Open
-
+	const { colorMode } = useColorMode();
 	return (
-		<Modal isOpen={isOpen} onClose={onClose}>
-			<ModalOverlay />
-			<ModalContent maxW={'700px'} borderWidth={3}>
-				<ModalHeader
-					bg={useColorModeValue('gray.200', 'gray.600')}
-					roundedTop="md"
-				>
-					<BountyHeader bounty={bounty} />
-				</ModalHeader>
-				<ModalCloseButton />
-				<ModalBody>
-					<BountyDetails bounty={bounty} />
-				</ModalBody>
-				<ModalFooter>
-					<BountyActions bounty={bounty} onCancel={onClose} />
-				</ModalFooter>
-			</ModalContent>
-		</Modal>
+		
+		<Center>
+			<Modal isOpen={isOpen} onClose={onClose}>
```

Comment:
If the modal is long and causes scrolling, the scrollbar is way on the right (the page itself appears to scroll). Shouldn't the scrollbar be on the right side of the modal itself?

Reply:
I will fix them now